mattermost4j-core/src/test/java/net/bis5/mattermost/client4/HttpConfigTest.java

```diff
@@ -0,0 +1,53 @@
+package net.bis5.mattermost.client4;
```

Comment:
Please insert license header comment like this.
https://github.com/maruTA-bis5/mattermost4j/blob/12127583e95da22263d758a35f97a7b05d8d6efb/mattermost4j-core/src/main/java/net/bis5/mattermost/client4/MattermostClient.java#L1-L15

Reply:
I have done that.

Thanks!